### ISSUE TITLE
Unable to install the plugin

### ISSUE THREAD

**Issue by nicolaslesquoy** [amtoine/nu_plugin_len#1]
*Description*

Compilation error when trying to install the plugin following the installation procedure described in `README.md`.

Here is the error message:
```
error: failed to compile `nu_plugin_len v0.1.0 (/home/nlesquoy/Documents/Projets/nu_plugin_hash/nu_plugin_len)`, intermediate artifacts can be found at `/home/nlesquoy/Documents/Projets/nu_plugin_hash/nu_plugin_len/target`

Caused by:
  failed to get `nu-plugin` as a dependency of package `nu_plugin_len v0.1.0 (/home/nlesquoy/Documents/Projets/nu_plugin_hash/nu_plugin_len)`

Caused by:
  failed to load source for dependency `nu-plugin`
...
```

---

*Steps to reproduce*

- Clone the repo in `nu_plugin_len`
- Run `cargo install --path .`

**Comment by amtoine** [comment]
yup, that's a very good catch :+1: 

actually there are already all the tool in the repo to fix this, but the documentation is not good enough :thinking: 

### i'll fix that right now and explain all this in details in the new documentation

**TL;DR** `nu-plugin` and `nu-protocol` are not in *crates.io* so you have to give a path to the source of `nushell`. this path is hardcoded to match my system. it does not work on yours 'cause `cargo install --path .` does not find the crates where they are specified in the `Cargo.toml`

**Comment by amtoine** [comment]
@nicolaslesquoy 

could you check #2 and tell me if that's any better? :relieved:
